packages/experimental/Divider/src/Divider.tsx

```diff
@@ -0,0 +1,76 @@
+/** @jsx withSlots */
+/** @jsxFrag */
+import React from 'react';
+import { View, ViewProps } from 'react-native';
+import { dividerName, DividerProps, DividerTokens } from './Divider.types';
+import { withSlots, compressible, UseTokens, useSlot, useFluentTheme, patchTokens, mergeStyles } from '@fluentui-react-native/framework';
+import { Text, TextProps } from '@fluentui-react-native/text';
+import { IconV1 as Icon, IconPropsV1 as IconProps } from '@fluentui-react-native/icon';
+import { useDividerTokens } from './DividerTokens';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import { colorsFromAppearance, useDividerSlotProps } from './Divider.styling';
+
+export const Divider = compressible<DividerProps, DividerTokens>((props: DividerProps, useTokens: UseTokens<DividerTokens>) => {
+  const { alignContent = 'center', appearance = 'default', color, icon, insetSize = 0, vertical = false } = props;
+
+  const theme = useFluentTheme();
+  let [tokens, cache] = useTokens(theme);
+
+  // call patch function to manually change token values from props (this is done by the lookup function in compose components)
```

Comment:
This comment feels a little unclear to me? IIRC, We want access to tokens inside our render, hence we need this call to patchTokens. Could we change the wording a bit to make that more clear?

This flow is an issue we had with other components (Shimmer, ActivityIndicator) so it would be nice to have a single documented solution for how to get around it :). Also I guess @chiuam / @ayush547 / @lenahong FYI Since y'all have / are working on Shimmer and Spinner.

Reply:
I'll address this when I work on Spinner